docs/DEVELOPER_NODE.md

```diff
@@ -0,0 +1,100 @@
+# AlastriaID Developer Node
+
+Alastria provides a node in the T (GoQuorum, IBFT 1.0) and B (BESU, IBTF 2.0) networks for those developers who want to contribute to the AlastriaID project.
+
+Access to this node is not open: access must be requested through this [form](https://docs.google.com/forms/d/e/1FAIpQLSdtixEkTFZKSMrmfn68qJP2UXrqCwyQGyb3Nx92Jqajy8yzMA/viewform).
+
+This guide describes how to access this resource:
+
+## Metamask
+
+Metamask is a very popular wallet that can be used for managing Ethereum accounts in a simple way.
+
+The connection with the nodes is made through the following URLs, using the access APIKEY:
+
+* AlastriaT Network:
+![AlastriaT Network](./img/alastria-t-metamask.jpg "AlastriaT Network")
+
+* AlastriaB Network:
+![AlastriaB Network](./img/alastria-b-metamask.jpg "AlastriaB Network")
+
+### Example
+
+![AlastriaB Network](./img/alastria-b-metamask-ok-0x50f606eabc06443cff710f9110d6c8ece8fd23206ac2d8adbcc6603d5828c204.jpg "AlastriaB Network")
+
+## Remix
+
+Remix is a well known development environment for Solidity that works both in the browser and on the desktop.
+
+The connection to the nodes can be made directly to the RPC service (http) of the node, or through Metamask
+
+There are some settings to consider for a successful connection:
+
+* Set EVM version (only for T Network):
+![AlastriaT Network](./img/alastria-t-remix-version.jpg "AlastriaT Network")
+
+* Use HTTP as Web3 provider:
+![AlastriaB Network](./img/alastria-t-remix-http-provider.jpg "HTTP provider")
+
+* Or use Metamask as provider:
+![Metamask provider](./img/alastria-t-remix-deploy-metamask.jpg "Metamask provider")
+
+## Truffle
+
+The connection to the node through Truffle is similar to those described above.
+
+These examples are based on the repository https://github.com/alastria/smart-contract-deployment:
+
+```
```

Comment:
I think it would be a good idea to put not only the configuration of the environment variables, if not indicate what changes to make in the truffle.js so that when running `truffle migrate --network alastriat` you get the IP of the node of the environment variable instead of the truffle.js. Don't you think @alejandroalffer ?